std/exception.d

```diff
@@ -1478,6 +1478,34 @@ private bool isUnionAliasedImpl(T)(size_t offset)
         static assert( isUnionAliased!(S.A5, 1)); //a5.b1;
 }
 
+package string errnoString(int errno)
```

Comment:
The inference is `nothrow @system`. I think we should mark it as `@trusted`, as it is a building block for `@safe` functions. I considered using `@trusted` lambdas, but there's not point, since almost every every statement is `@system` and this would make the function harder to read. Plus, it's small enough that you can review it at a glance.